### DISCUSSION TITLE
Nanojsx plus unocss or suggest a stack for a nanoframework?

### DISCUSSION THREAD

**Discussion by solarsystemic** [nanojsx/nano#103] [General]
Nanojsx sounds incredible.  I found unocss - https://github.com/unocss/unocss  for using extremely fast css. I found it in Deno's list of approved 2000 or so libraries. Then it disappeared! So maybe its not "approved"... It sounds perfect to pair with Nanojsx because its "nano" size and is extremely ....fast.  Like another poster I have never used javascript but I will need shortly to use client and server code for a forum. I can't use any of the usual code from say php or python.  What I need and probably many other non Javascript coders is a nano framework or just a few right libraries. For me I am finishing up with a JS stack but its quite large and a typical stack. Vue with Deno and the usual thing.

What would you use a "stack" for Nanojsx for sure, Unocss and whatever?

To me there is a huge need for non javascript coders where a nano stack makes things easy and fast but perhaps simple.....  Just something like the usual buttons, drop down windows but simple......

I saw your first screens showing your controls. That is perfect.  Almost there for a "nano-framework".

Here is this youtube about unocss.  https://www.youtube.com/watch?v=yZnwzSin7Zc

**Comment by pierredewilde** [comment]
You may combine [nanojsx](https://nanojsx.io/) with [Twind](https://twind.dev/) using [nanossr](https://crux.land/nanossr@0.0.5) as explained in Deno Blog post [A Whole Website in a Single JavaScript File](https://deno.com/blog/a-whole-website-in-a-single-js-file-continued).

Twind is also used by [Fresh](https://fresh.deno.dev/).

Or you may easily integrate [UnoCSS](https://github.com/unocss/unocss) with any framework with [UnoCSS CLI](https://www.npmjs.com/package/@unocss/cli).

UnoCSS CLI quickly scans your source files and generates a `uno.css` file to be loaded in your HTML files.
